### PR TITLE
Audio Source AM/FM/XM/DAB

### PR DESCRIPTION
**Pull request by ValeriiMalkov**
### Audio Source AM/FM/XM/DAB

[Proposal](https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0182-audio-source-am-fm-xm.md)

### Summary

Add request with `moduletype=AUDIO`
for seting source `AM/FM/XM`.

### CODE REVIEW

File: app/model/media/RadioModel.js [hunk 1]
```diff
@@ -638,6 +638,30 @@ SDL.RadioModel = Em.Object.create({
...
 
+  checkoutRadioSource: function(data){
+    if(data.source == 'AM' || 
+        data.source == 'FM' || 
+          data.source == 'XM'){
+      
+      if (data.source != this.radioControlStruct.band) {
+        SDL.RadioModel.setRadioBand(data.source);
+
+        if (data.source == 'FM') {
```
Comment (AKalinich-Luxoft) [inline app/model/media/RadioModel.js]: @ValeriiMalkov all these three `if` could be merged:
```
if (data.source == 'AM' || data.source == 'FM' || data.source == 'XM')
```

File: app/model/media/RadioModel.js [hunk 2]
```diff
@@ -1466,12 +1490,12 @@ SDL.RadioModel = Em.Object.create({
...
     var data = SDL.MediaController.getAudioControlData();
     if(data){
-    FFW.RC.onInteriorVehicleDataNotification({moduleType:'AUDIO',audioControlData: {'source':data.source}});
+    FFW.RC.onInteriorVehicleDataNotification({moduleType:'AUDIO',audioControlData: {'source':this.lastOptionParams.band}});
   }
   },
   setSource:function()
   {
-    SDL.MediaController.lastRadioControlStruct.source='RADIO_TUNER';
+    SDL.MediaController.lastRadioControlStruct.source=this.lastOptionParams.band;
```
Comment (AKalinich-Luxoft) [inline app/model/media/RadioModel.js]: @ValeriiMalkov add spaces please

File: app/model/media/RadioModel.js (hunk 1 above)
Comment (ValeriiMalkov) [reply app/model/media/RadioModel.js]: @AKalinich-Luxoft I think i can delete this case. See that in commit b4e087f

File: app/model/media/RadioModel.js (hunk 2 above)
Comment (ValeriiMalkov) [reply app/model/media/RadioModel.js]: @AKalinich-Luxoft Fixed in commit b4e087f

**Pull request merged**